services/storage/tests/unit/test_modules_celery.py

```diff
@@ -99,21 +99,20 @@ def dreamer_task(task: AbortableTask) -> list[int]:
 @pytest.fixture
 def register_celery_tasks() -> Callable[[Celery], None]:
     def _(celery_app: Celery) -> None:
-        define_task(celery_app, sync_archive)
-        define_task(celery_app, failure_task)
-        define_task(celery_app, dreamer_task)
+        register_task(celery_app, fake_file_processor)
```

Comment:
suggestion: one of these functions should be made async to test also the async use case as the register_task handles both cases.